schema.sql

```diff
@@ -1,5 +1,11 @@
 /* Database schema to keep the structure of entire database. */
 
-CREATE TABLE animals (
-    name varchar(100)
+CREATE TABLE animals(
+  id integer,
```

Comment:
[OPTIONAL] Please mark the `id` column as the primary key and make the `id` column not null and autoincremented :slightly_smiling_face:

Reply:
Thank you. I will do that.